pages/reference/hardware/versioning.md

```diff
@@ -37,14 +37,30 @@ The community devices types are:
 - Updated to newer versions as the community drives it, we do not guarantee the device type will keep up with OS versions.
 - Community boards can be promoted to an official balena supported board if a customer is interested in sponsoring the device support. If you are interested in sponsoring a community board, email solutions@balena.io .
 
-#### Discontinued
-Discontinued device types are those that are no longer actively maintained by the balena team or anyone in the community and will stop showing in the dashboard as options for new fleets and devices. Any already provisioned devices of this type will continue to function, but will no longer receive operating system updates.
-
-A device type will be discontinued if:
-- The device manufacturer end support and sale of the device.
-- No devices of this type are provisioned on balenaCloud in the last 6 months.
-- If it is a community device type and it has not received maintenance in the last 6 months.
-- If it is a private device type and the customer has stopped paying for the device to be supported.
+#### Device Type support policy
+At balena, we’re here to make sure balenaOS runs smoothly on actively maintained hardware models. This policy lays out what we support, what you can expect, and how we handle devices when they reach the end of their lifecycle.
+
+**What We Offer for Official Supported Hardware Models**
+- balenaOS Image: A tested and ready-to-use version of balenaOS, available in our documentation.
+- Incident and Bug Support: We’ll help troubleshoot issues with incidents, bugs, or performance—whenever possible.
+- Regular Updates: You’ll get ongoing updates and patches to keep your devices secure, compatible, and running efficiently.
+
+**What Happens When a Device is Discontinued?**
+Sometimes, a device type reaches the end of the road. When this happens:
+- The device type will stop appearing as an option for creating new fleets or adding new devices in the dashboard.
+- Any devices already provisioned will keep working, but they won’t get any more OS updates.
+
+**When Does a Device Get Discontinued?**
```

Comment:
I don't think we currently discontinue devices from BC - we commit to keeping them working as they were on the last release. That is made possible by API versioning and the introduction of new features on new versioned API. This is obviously a great for the product as a value add for customers, and up to date we have been able to afaik, but it may be something the backend guys object to.